src/test/java/com/aleksey5678/uniquecharacterscounter/stringformatter/MapToStringFormatterTest.java

```diff
@@ -0,0 +1,34 @@
+package com.aleksey5678.uniquecharacterscounter.stringformatter;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class MapToStringFormatterTest {
```

Comment:
please also add test case for empty string `""`

Reply:
Done